src/style.css

```diff
@@ -1 +1,102 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
+  padding: 0;
+  margin: 0;
```

Comment:
let's not reset padding and margin in this tag
you have already done it for margin in `body` tag